### ISSUE TITLE
[Bug]: BVT  index out of range

### ISSUE THREAD

**Issue by zhangxu19830126** [matrixorigin/matrixone#8570]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):main
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1386" alt="image" src="https://user-images.githubusercontent.com/2995754/226519080-1e4d098c-08c2-45ea-b52f-41cf26e4378b.png">

**Comment by nnsgmsone** [comment]
i will fixed it

**Comment by nnsgmsone** [comment]
will fixed by #8667
